modules/op-tkn-pipelines-compatibility-support-matrix.adoc

```diff
@@ -0,0 +1,35 @@
+[id="compatibility-support-matrix_{context}"]
+= Compatibility and support matrix
+
+Some features in this release are currently in link:https://access.redhat.com/support/offerings/techpreview[Technology Preview]. These experimental features are not intended for production use.
+
+In the table, features are marked with the following statuses:
+
+[horizontal]
+TP:: Technology Preview
+GA:: General Availability
+
+.Compatibility and support matrix
+[options="header"]
+|===
+
+| {pipelines-title} Version 4+| Component Version | OpenShift Version | Support Status
+
+|Operator | Pipelines | Triggers | CLI | Catalog | |
+
+|1.0 | 0.11.3 | 0.4.0       | 0.9    | 0.11 | 4.4 | TP
```

Comment:
Suggest reversing the order - 1.6 on the first row and 1.0 on the last, this will make it consistent with the placement of our release modules and place the information about the latest release right at the top.

Reply:
Updated. Thanks.